src/software/simulated_tests/simulated_test_fixture.cpp

```diff
@@ -27,7 +27,6 @@ void SimulatedTestFixture::SetUp()
     // guarantee the pointer will never be null / empty
     simulator = std::make_unique<Simulator>(Field::createSSLDivisionBField());
     ai = AI(DynamicParameters->getAIConfig(), DynamicParameters->getAIControlConfig());
-    sensor_fusion = SensorFusion(DynamicParameters->getSensorFusionConfig());
```

Comment:
why was this deleted?

Reply:
Due to the mutex added to SensorFusion, it no longer has a copy constructor so has to be created in the tests initialization list, and can't be copied in the setup here.